docs/tests.md

```diff
@@ -0,0 +1,18 @@
+# Tests for actors
+
+You can implement two types of tests for an actor in the leapp-repository - functional and unit tests.
+
+## Functional tests
+   - These tests provide fabricated input messages for the actor, check the output messages and check for side effects stated in the actor's description.
```

Comment:
From @amahdal: "check the output messages and check for side effects stated in the actor's description" .. side effect is something else, e.g. processor overheating. Mention just "check the outputs stated in the actor's description".